cmd/deploy/deploy.go

```diff
@@ -148,7 +148,7 @@ func Deploy(ctx context.Context) *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to get the current working directory: %w", err)
 			}
-			name := ""
+			name := options.Name
```

Comment:
If `options.Name` was being set, we we generating Kubefiles called `kubeconfig-` as name was empty